mobile/library/swift/EngineBuilder.swift

```diff
@@ -546,6 +557,74 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Sets the node.id field in the Bootstrap configuration.
+  ///
+  /// - parameter nodeId: The node ID.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setNodeId(_ nodeId: String) -> Self {
```

Comment:
Nit: We should follow [Swift's API naming guidelines](https://www.swift.org/documentation/api-design-guidelines/) here

```suggestion
  public func setNodeID(_ nodeID: String) -> Self {
```

> [Acronyms and initialisms](https://en.wikipedia.org/wiki/Acronym) that commonly appear as all upper case in American English should be uniformly up- or down-cased according to case conventions:
> ```swift
> var utf8Bytes: [UTF8.CodeUnit]
> var isRepresentableAsASCII = true
> var userSMTPServer: SecureSMTPServer
> ```